src/txmempool.cpp

```diff
@@ -422,8 +422,6 @@ void CTxMemPool::addUnchecked(const CTxMemPoolEntry &entry, setEntries &setAnces
     indexed_transaction_set::iterator newit = mapTx.insert(entry).first;
 
     // Update transaction for any feeDelta created by PrioritiseTransaction
-    // TODO: refactor so that the fee delta is calculated before inserting
-    // into mapTx.
     CAmount delta{0};
     ApplyDelta(entry.GetTx().GetHash(), delta);
```

Comment:
Don't have much background on what was intended, but I interpret this TODO to be "move these 2 lines up before the `insert` call, and delete the `mapTx.modify()` call below?

Reply:
Yes, it is possible to fix it in that way, see commit fa300f581fe6cd4b37c7cefb191722ff970e0116. Are you suggesting I open a pull with that?

Reply:
Yeah I think it's better, regardless of whether that's what the todo is referring to

Reply:
Should we just convert this PR into the suggested fix?